src/lib.rs

```diff
@@ -24,8 +24,7 @@
 //! # Example - Running a VM on x86_64
 //!
 //! In this example we are creating a Virtual Machine (VM) with one vCPU.
-//! On the vCPU we are running x86_64 specific code. This example is based on
```

Comment:
I would still leave the LWN article as a reference here. Even though the arm code is different, we still follow the workflow as described in that article. We can say that for aarch64 the code is adapted to work on that platform.